src/app/containers/Ad/__snapshots__/index.test.jsx.snap

```diff
@@ -1,67 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Ad Container Snapshots should correctly render an AMP ad 1`] = `
-.c0 {
-  margin: 0 auto;
-  width: 100%;
-  text-align: center;
-}
-
-@media (min-width:37.5rem) {
-  .c0 {
-    margin-top: 1.5rem;
-  }
-}
-
-@media (min-width:63rem) {
-  .c0 {
-    margin-top: 2rem;
-    max-width: 63rem;
-  }
-}
-
-<div
-  class="c0"
->
-  <amp-ad
-    data-a4a-upgrade-type="amp-ad-network-doubleclick-impl"
-    data-amp-slot-index="0"
-    data-multi-size="320x50,300x50"
-    data-slot="/4817/bbccom.test.site.amp.news"
-    height="50"
-    json="{\\"targeting\\":{\\"slot\\":\\"leaderboard\\",\\"asset_type\\":\\"index\\",\\"channel\\":\\"pidgin\\"}}"
-    media="(max-width: 599px)"
-    type="doubleclick"
-    width="320"
-  >
-    <amp-img
-      height="50"
-      layout="responsive"
-      placeholder="true"
-      src="https://via.placeholder.com/320x50"
-      width="320"
-    />
-  </amp-ad>
-  <amp-ad
-    data-a4a-upgrade-type="amp-ad-network-doubleclick-impl"
-    data-amp-slot-index="0"
-    data-multi-size="970x250,728x90"
-    data-slot="/4817/bbccom.test.site.amp.news"
-    height="250"
-    json="{\\"targeting\\":{\\"slot\\":\\"leaderboard\\",\\"asset_type\\":\\"index\\",\\"channel\\":\\"pidgin\\"}}"
-    media="(min-width: 600px)"
-    type="doubleclick"
-    width="970"
-  >
-    <amp-img
-      height="250"
-      layout="responsive"
-      placeholder="true"
-      src="https://via.placeholder.com/970x250"
-      width="970"
-    />
-  </amp-ad>
-</div>
-`;
+exports[`Ad Container Snapshots should correctly render an AMP ad 1`] = `<div />`;
```

Comment:
This should be rendering something - let's make sure the ads toggle is set to true for this test